.github/workflows/update-project.yml

```diff
@@ -12,6 +12,9 @@ env:
   PR_STATUS_DRAFT: 'In Progress'
   PR_STATUS_READY: 'Review/QA'
 
+permissions:
+  pull-requests: write
```

Comment:
would it be good to add the `repository-projects` here?
It may not matter much sinc eth eproject is at the org level (I believe) and that's why we need the zowe-robot-token to have the right permissions 😋

Reply:
Yeah, I was thinking the same thing. Since it is an org project, we are using the Zowe Robot token, so it shouldn't matter.